crates/theme_selector/src/theme_selector.rs

```diff
@@ -66,6 +55,8 @@ fn toggle_icon_theme_selector(
     });
 }
 
+
```

Comment:
There are issues with formatting on CI, which we might want to fix with `cargo fmt --all` + removing a few extra blank lines added.